src/components/ConfirmationBox/ConfirmationBox.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {shallow, configure} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import ConfirmationBox from './ConfirmationBox';
+
+configure({adapter: new Adapter()});
+
+it('calls the correct functions when clicked', () => {
+	const testFuncYes = jest.fn();
+	const testFuncNo = jest.fn();
```

Comment:
nit pick (feel free to ignore)
this is like a mock so would be better names mockOnYes or just handleYes

Reply:
changed